dash-pipeline/SAI/templates/utils.cpp.j2

```diff
@@ -217,29 +217,155 @@ sai_status_t sai_create_switch_dummy(
     return SAI_STATUS_SUCCESS;
 }
 
-sai_status_t sai_get_switch_attribute_dummy(
+#define DASH_BMV2_NUM_PORTS 2 
+#define DASH_BMV2_CPU_QOS_NUMBER_OF_QUEUES 0
+
+#define OJBTYPE_SHFT 48
+#define MAKE_OID(_objtype, _objval) (sai_object_id_t)(((sai_object_id_t)_objtype<<OJBTYPE_SHFT)+(sai_object_id_t)_objval)
+
+#define DASH_BMV2_DEFAULT_CPU_PORT_ID MAKE_OID(SAI_OBJECT_TYPE_PORT,64)
+#define DASH_BMV2_DEFAULT_VLAN_ID MAKE_OID(SAI_OBJECT_TYPE_VLAN,1)
+#define DASH_BMV2_DEFAULT_VRF_ID MAKE_OID(SAI_OBJECT_TYPE_VIRTUAL_ROUTER,1)
+#define DASH_BMV2_DEFAULT_1Q_BRIDGE_ID MAKE_OID(SAI_OBJECT_TYPE_BRIDGE,1)
+#define DASH_BMV2_DEFAULT_1Q_BRIDGE_ID MAKE_OID(SAI_OBJECT_TYPE_BRIDGE,1)
+
+sai_object_id_t port_list[DASH_BMV2_NUM_PORTS] = {
+    MAKE_OID(SAI_OBJECT_TYPE_PORT,1),
+    MAKE_OID(SAI_OBJECT_TYPE_PORT,2)
+};
+
+sai_status_t sai_get_switch_attribute(
```

Comment:
can we move this to saiswitch.cpp?

Reply:
I'll give it a try,thanks.

Reply:
I created a `saifixedapis.cpp.j2` file which contains a mix of API callbacks, not just switch-related. If this grows too large we can split off into more files. I'm trying to strike a balance between modularity and file bloat.